builder/amazon/common/step_ami_region_copy.go

```diff
@@ -64,7 +64,7 @@ func (s *StepAMIRegionCopy) Run(ctx context.Context, state multistep.StateBag) m
 	ui := state.Get("ui").(packer.Ui)
 	amis := state.Get("amis").(map[string]string)
 	snapshots := state.Get("snapshots").(map[string][]string)
-	intermediary := state.Get("intermediary_image").(bool)
+	intermediary, _ := state.Get("intermediary_image").(bool)
```

Comment:
```suggestion
	intermediary, ok := state.GetOk("intermediary_image").(bool)
	if !ok {
	    intermediary = false
	}
```

Reply:
Hmm on second glance, I don't think this alone is gonna be enough. Did you build with this change?